cpp/include/raft/spatial/knn/detail/ivf_pq_search.cuh

```diff
@@ -405,23 +405,42 @@ void postprocess_distances(float* out,        // [n_queries, topk]
                            distance::DistanceType metric,
                            uint32_t n_queries,
                            uint32_t topk,
+                           double scaling_factor,
```

Comment:
Would you consider using `float` here and in the other signatures? We don't seem to be gaining anything by passing this factor as `double` here, since in all cases it's immediately casted to `float` anyway.